lib/collections.js

```diff
@@ -47,6 +65,21 @@ longAndShort['longest'] = '';
 longAndShort['shortest'] = paragraph;
 
 /* your code starts here */
+var longest = " ";
+var shortest = " ";
+
+//Going through the array of unique words to determine the longest and shortest. Variables carry out a duty of a buffer for storing the longest and shortest words.
+for (var i = 1; i < uniqueWords.length; i++) {
+  if (longest.length < uniqueWords[i].length) {
+    longest = uniqueWords[i];
+  }
+  if (shortest.length >= uniqueWords[i].length) {
+    shortest = uniqueWords[i];
+  }
+}
+
+longAndShort.longest = longest;
+longAndShort.shortest = shortest;
```

Comment:
Rather than creating those variables `longest` and `shortest`, it might have been simpler to just use the `longAndShort` object that Antony gave you.